From afe3180643aff625ea327fd01861df723da63b4f Mon Sep 17 00:00:00 2001
From: Lin Jinhan <troy.lin@rock-chips.com>
Date: Fri, 27 Sep 2019 15:37:54 +0800
Subject: [PATCH] arm64: configs: rockchip_defconfig: enable crypto rockchip

enable CONFIG_CRYPTO_DEV_ROCKCHIP_V1
enable CONFIG_CRYPTO_DEV_ROCKCHIP_V2

Change-Id: Idd4a8e0df284348980c54e7bdc36324f5053e19a
Signed-off-by: Lin Jinhan <troy.lin@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 74e72d8ca36e..98b4dcf2db71 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -847,6 +847,8 @@ CONFIG_CRYPTO_SHA512=y
 CONFIG_CRYPTO_TWOFISH=y
 CONFIG_CRYPTO_LZ4=y
 CONFIG_CRYPTO_ANSI_CPRNG=y
+CONFIG_CRYPTO_DEV_ROCKCHIP_V1=y
+CONFIG_CRYPTO_DEV_ROCKCHIP_V2=y
 CONFIG_ARM64_CRYPTO=y
 CONFIG_CRYPTO_SHA1_ARM64_CE=y
 CONFIG_CRYPTO_SHA2_ARM64_CE=y
-- 
2.35.3

